micropython/micropython-1.15-config/patches/10_py_mkrules.patch

```diff
@@ -0,0 +1,16 @@
+diff -Naur micropython-1.15/py/mkrules.mk micropython-phoenix/py/mkrules.mk
+--- micropython-1.15/py/mkrules.mk	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/py/mkrules.mk	2021-06-30 11:48:28.976939829 +0200
+@@ -184,7 +184,11 @@
+ 	$(ECHO) "LINK $@"
+ # Do not pass COPT here - it's *C* compiler optimizations. For example,
+ # we may want to compile using Thumb, but link with non-Thumb libc.
+-	$(Q)$(CC) -o $@ $^ $(LIB) $(LDFLAGS)
++ifndef PHOENIX_PORT_LDFLAGS
```

Comment:
should not be needed, I've checked that adding `-nostartfiles` to LDFLAGS solves for now the `R_ARM_ABS32` relocations in `.rel.text` (but that should be added as a temporary stop-gap with FIXME until it would be fixed in toolchain).

Moreover: our make-exported LDFLAGS are in `LD` format, it would be nice to be able to translate it in some generic way (for rt1064 we should get something like `-Wl,-z,max-page-size=0x10 -Wl,-q `)